tutorials/plot_sensors_time_frequency.py

```diff
@@ -87,7 +87,8 @@
 # but you can also use :func:`mne.time_frequency.tfr_multitaper`
 # or :func:`mne.time_frequency.tfr_stockwell`.
 
-freqs = np.arange(6, 30, 3)  # define frequencies of interest
+# define frequencies of interest (log-spaced)
+freqs = np.logspace(*np.log10([6, 35]), num=8)
```

Comment:
The y-axis limits don't look great to me here:

https://3531-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_sensors_time_frequency.html

It seems like rather than ending at the frequency limits, they should end at the `pcolormesh` boundaries beyond those. It makes it clearer it's an imshow-like plot.